components/BCOnlineAccount.vue

```diff
@@ -1,143 +1,153 @@
-              <v-spacer></v-spacer>
-            </v-card-title>
-            <v-card-text class="menu-item pl-7 pr-7">
-              <p class="ma-0 pt-8 pb-6">
-                File a Wills Notice or search for an existing Wills Notice.
-                <span style="font-weight: bold"
-                  >Wills Registry will also be available through the BC
-                  Registries Account soon.</span
-                >
-              </p>
-            </v-card-text>
-          </v-card>
-          <v-card
-            elevation="2"
-            max-width="375px"
-            class="mr-5 mb-8 justify-center"
-          >
-            <v-card-title class="menu-header">
-              <span class="menu-item-title">Address Change BC</span>
-              <v-spacer></v-spacer>
-            </v-card-title>
-            <v-card-text class="menu-item pl-7 pr-7">
-              <p class="ma-0 pt-8 pb-6">
-                Update your personal and/or business records with provincial and
-                local government organizations.
-              </p>
-            </v-card-text>
-          </v-card>
+          <v-card-title class="menu-header">
+            <span class="menu-item-title">Address Change BC</span>
+            <v-spacer></v-spacer>
+          </v-card-title>
+          <v-card-text class="menu-item pl-7 pr-7">
+            <p class="ma-0 pt-8 pb-6">
+              Update your personal and/or business records with provincial and
+              local government organizations.
+            </p>
+          </v-card-text>
         </v-card>
-        <div class="payment ma-0 pt-7 pb-7 mr-10">
-          <v-row no-gutters class="ma-0 pa-0">
-            <v-col cols="2"></v-col>
-            <v-col cols="8" class="payment-info-bold"
-              ><v-icon medium color="#38598a">mdi-currency-usd-circle</v-icon
-              ><span style="padding-left: 10px">BC OnLine payments</span></v-col
-            >
-            <v-col cols="2"> </v-col>
-          </v-row>
-          <v-row no-gutters class="ma-0 pa-0">
-            <v-col cols="3"></v-col>
-            <v-col cols="6" class="payment-info" style="max-width: 850px"
-              >All payments for BC OnLine accounts are made using pre-authorized
-              BC OnLine account payments.
-            </v-col>
-            <v-col cols="3"></v-col>
-          </v-row>
-          <v-row class="button-row">
-            <v-col cols="12">
-              <v-btn class="button-blue" :href="$config.bcolURL"
-                >Log in to my BC OnLine Account</v-btn
-              >
-            </v-col>
-          </v-row>
-        </div>
-      </v-container>
-    </div>
+      </v-card>
+
+      <div class="payment pt-7 pb-7">
+        <v-row no-gutters class="ma-0 pa-0">
+          <v-col cols="2"></v-col>
+          <v-col cols="8" class="payment-info-bold">
+            <!-- TODO: the following should be a "circle" icon -- not avail in MDI, use SVG instead -->
```

Comment:
Waiting for Tracey to provide this. In the meantime, the specified icon below works.